Makefile

```
@@ -35,6 +35,14 @@ dev:
 		go install -ldflags "${LD_FLAGS}" -tags "${GOTAGS}"
 .PHONY: dev
 
+# dev docker builds
+docker:
+	@env CGO_ENABLED="0" go build -ldflags "${LD_FLAGS}" -o $(NAME)
+	mkdir -p dist/linux/amd64/
+	cp consul-esm dist/linux/amd64/
+	env DOCKER_BUILDKIT=1 docker build -t consul-esm .
```

Minor, just saw that `$(NAME)` wasn't being used consistently and this could be a line shorter.
```suggestion
	mkdir -p dist/linux/amd64/
	@env CGO_ENABLED="0" go build -ldflags "${LD_FLAGS}" -o dist/linux/amd64/$(NAME)
	env DOCKER_BUILDKIT=1 docker build -t $(NAME) .
```